docs/content/_navigation.json

```diff
@@ -399,6 +399,10 @@
           {
             "title": "Re-execution",
             "path": "/guides/dagster/re-execution"
+          },
+          {
+            "title": "Example Repository",
```

Comment:
Is "repository" the right word here?  Is it referring to repository in the Dagster sense or in the git sense?  If, in the Dagster sense, it might become confusing when we convert to graphs/jobs and end up having multiple repositories?  Maybe "project" would make more sense?

I think it would also be useful to give some sort of adjective that describes the role of this example.  We have lots of examples in different places - what's unique about this one is that it's meant to be realistic/maximalist/holistic/full-fledged/complete.